js/events/index.d.ts

```diff
@@ -6,6 +10,50 @@ export interface EventType { }
  */
 export type TEvent = {} extends EventType ? dxEvent : EventType;
 
+/** @public */
+export interface ComponentEvent<T> {
```

Comment:
These events are the base part for almost all events. Could you check if `readonly` modifier is applicable?
- `ComponentEvent`
- `ComponentNativeEvent`
- `ComponentDisposingEvent`
- `ComponentInitializedEvent`
- `ComponentOptionChangedEvent`
/cc @DokaRus, @IgnatovDan

Reply:
https://github.com/IlyaKhD/DevExtreme/pull/1/files : component/element/model members are readonly in event args